addons/l10n_es_edi_facturae/tests/test_edi_xml.py

```diff
@@ -125,7 +136,11 @@ def test_cannot_generate_unsigned_xml(self):
                 patch(f"{self.certificate_module}.sha1", lambda x: sha1()):
             invoice = self.create_invoice(partner_id=self.partner_a.id, move_type='out_invoice', invoice_line_ids=[{'price_unit': 100.0, 'tax_ids': [self.tax.id]},],)
             invoice.action_post()
-            self.assertRaises(UserError, invoice._l10n_es_edi_facturae_render_facturae)
+            generated_file, errors = invoice._l10n_es_edi_facturae_render_facturae()
+            if generated_file is not None and errors:
```

Comment:
Why the `if`?
This would just fail at failing if there is an error...

Reply:
Yes, you are right!

Code updated!